test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -178,15 +181,15 @@ def submit_job(self, sleep=10, lower=0,
             if job_id is not None:
                 self.assertEqual(jid.split('.')[0], job_id)
             if arr_flag:
-                self.server.expect(JOB, {'job_state': 'B'}, id=jid)
-                self.server.expect(JOB, {'job_state=R': '%d' % (total_jobs)},
-                                   count=True, id=jid, extend='t')
-                if sleep == 1:
-                    self.server.expect(JOB, 'queue', op=UNSET, id=jid)
+                if status is not None:
+                    self.server.expect(JOB, {'job_state': 'B'}, id=jid)
+                    self.server.expect(
+                        JOB,
+                        {'job_state=R': '%d' % (total_jobs)},
```

Comment:
Can you not write total_jobs to the dictionary instead of substituting it as a string?

Reply:
Addressed!